lib/beaker/host/windows.rb

```diff
@@ -58,17 +58,10 @@ def self.aio_defaults
       h = Beaker::Options::OptionsHash.new
       h.merge({
         'user'              => 'Administrator',
-        'group'             => 'Administrators',
         # 'puppetserver-confdir' not applicable on Windows
         # 'puppetservice' not applicable on Windows
-        'puppetpath'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetconfdir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetvardir'      => '`cygpath -smF 35`/PuppetLabs/puppet/cache',
-        # 'puppetbin' not required on Windows
-        'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
-        # systembindir not required on Windows
-        'hieradatadir'      => '`cygpath -smF 35`/Puppetlabs/code/hieradata',
-        'hieraconf'         => '`cygpath -smF 35`/Puppetlabs/code/hiera.yaml',
+        'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin',
+        'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
```

Comment:
Because it's interpreted by cygwin's bash. When it executes a process, it converts `PATH=/cygdrive/c/dir1:/cygdrive/c/dir2` to `PATH=c:\dir1;c:\dir2`